docs/elements.md

```diff
@@ -1382,14 +1382,14 @@ Display a statically or dynamically generated image.
-| `type`      | text    | 'static'              | Type of file, either 'static' (an existing file) or 'dynamic' (a file generated by element or server code).                                                                                           |
-| `directory` | text    | "clientFilesQuestion" | The directory that contains the file, either 'clientFilesQuestion' or 'clientFilesCourse' (see [client and server files](clientServerFiles.md)). A directory cannot be specified if `type='dynamic'`. |
-| `width`     | number  | `None`                | Width of image (e.g., '250px').                                                                                                                                                                       |
-| `inline`    | boolean | false                 | Display figure inline with text (true) or on a separate line (false).                                                                                                                                 |
-| `alt`       | text    | ""                    | Provide alt (alternative) text to improve accessibility of figures by describing the image or the purpose of the image. Default is an empty string.                                                   |
+| Attribute   | Type    | Default                 | Description                                                                                                                                                                                             |
+| ----------- | ------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `file-name` | string  | —                       | Name of image file.                                                                                                                                                                                     |
+| `type`      | text    | `"static"`              | Type of file, either 'static' (an existing file) or 'dynamic' (a file generated by element or server code).                                                                                             |
+| `directory` | text    | `"clientFilesQuestion"` | The directory that contains the file, either `"clientFilesQuestion"` or `clientFilesCourse` (see [client and server files](clientServerFiles.md)). A directory cannot be specified if `type="dynamic"`. |
+| `width`     | number  | `"None"`                | Width of image (e.g., '250px').                                                                                                                                                                         |
```

Comment:
Not sure None should be in quotes here, the default in this case is to use the width of the source image file.